package.json

```diff
@@ -30,7 +30,7 @@
     "typescript": "^4.5.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
C8. remove `--openssl-legacy-provider`, it may not work for every OS so i recommend to remove it here but use it locally if it is really necessary